### PR TITLE
Adds FindLibP11.cmake

### PR DESCRIPTION
**Pull request by johnfxgalea-e**
Adds FindLibP11.cmake so that find_package() may be invoked for the LibP11 library.

### CODE REVIEW

**Review by garw — APPROVED** [review]
Looks roughly okay but this needs the be an optional dependency that can be disabled (or rather should be explicitly enabled). MoCoCrW needs to be compilable if there is no libp11 present on the system.

**Review by garw — CHANGES_REQUESTED** [review]
err… wrong button :/

**Comment by johnfxgalea-e** [comment]
> Looks roughly okay but this needs the be an optional dependency that can be disabled (or rather should be explicitly enabled). MoCoCrW needs to be compilable if there is no libp11 present on the system.

Done.

**Review by garw — APPROVED** [review]
lg2m

File: cmake/FindLibP11.cmake [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+# Configuration:
+#  LIBP11_ROOT_DIR     - Root directory containing the installation of LibP11.
+#                        Used as a hint to find the library.
+#
+# This module defines the following variables:
+#  LIBP11_FOUND        - If LibP11 is available
+#  LIBP11_INCLUDE_DIR  - The include directory of LibP11.
+#  LIBP11_LIBRARY      - The LibP11 Library.
+#
+# This module defines the following`IMPORTED` targets:
```
Comment (blenk92) [inline cmake/FindLibP11.cmake]: looks like there should be a white space between ``following`IMPORTED` ``

Comment (johnfxgalea-e) [reply cmake/FindLibP11.cmake]: Done.

**Review by blenk92 — APPROVED** [review]

**Pull request merged**